far/FarEng.hlf.m4

```diff
@@ -3434,6 +3434,12 @@ string is changed to that of the found string. For example:
 $ #Editor: All matching entries menu#
  The following key combinations are available in this menu:
 
+ #F2#
+ Copy all found lines into a new editor and close this menu.
```

Comment:
Unfortunately, it's not clear from this phrase what exactly will be copied to the new editor: found file paths (full), verbatim lines presented in the dialog (relative/truncated), or text lines from the found files' content.